### ISSUE TITLE
Nova slows down dramatically

### ISSUE THREAD

**Issue by holubar** [AlexAzartsev/heroicon#14]
I did not find better package for icons, thank you for great addon. However, when using it, Nova slows down. Every request -> index, edit or detail, where this field is used takes multiple seconds which becomes unusable.

Is there any space for performance optimisation?

**Comment by AlexAzartsev** [comment]
thx for feedback, I'll take a look into this problem, performance issues can depend on how many active icons sets enabled,
as a workaround you can registered your own icon set with limited set of icons, and disable fontawesome icons set. If this approach will suit your needs.

**Comment by scramatte** [comment]
Same issue here. I've literally using 10 icons. My question is: If svg data is stored into database why is so slow? Component shouldn't load everything on index. 

I'm going to try what you told above. But it will interesting to fix it in another way.

**Comment by AlexAzartsev** [comment]
Yep, performance issues appears because of rendering of all possible values for selection from icons set.
To be able to render them, I need to load it from a backend, now they all loaded at once.

Will work on improvement of this process, but for now, quick fix for you will be to create your custom set of used icons, and disable not used sets.
